gnocchi/rest/__init__.py

```diff
@@ -1649,6 +1649,9 @@ def get_cross_metric_measures_from_objs(metrics, start=None, stop=None,
                                             refresh=False, resample=None):
         try:
             needed_overlap = float(needed_overlap)
+            if needed_overlap != 100.0 and start is None and stop is None:
```

Comment:
Can you put that outside of the `try/except`? It should not be caught as part of this block if it'd ever raise a `ValueError` (unlikely now, but who knows in the future :))